routes/web.php

```diff
@@ -23,6 +23,23 @@
     return "submit successfully";
 })->name('login_accepted');
 
+Route::get('/cv', function () {
+    return view('cv');
+});
+
+#for task3 
+#the task is to send the form data to another page
+#and to show the data in a proper format
+#begin
+Route::get('/login_task3', function () {
+    return view('task3_login');
```

Comment:
again here,, make sure your naming is expressive, this is not a login page it is a contact us